mne/decoding/transformer.py

```diff
@@ -157,92 +159,122 @@ class EpochsVectorizer(TransformerMixin):
 
     Attributes
     ----------
-    n_channels : int
+    n_channels_ : int
         The number of channels.
-    n_times : int
-        The number of time points.
+    n_epochs_ : int
+        The number of data samples.
```

Comment:
Alright, got it. Although the number of channels in every dataset has to be the same. Is that correct?